webpack/assets/javascripts/react_app/components/common/forms/NumericInput.js

```diff
@@ -1,48 +1,48 @@
-import NumericInput from 'react-numeric-input';
+import ReactNumericInput from 'react-numeric-input';
 import React from 'react';
 import PropTypes from 'prop-types';
 
 import { noop } from '../../../common/helpers';
 import CommonForm from './CommonForm';
 
-const TextInput = ({
+const NumericInput = ({
   label,
   className,
   value,
-  onChange,
+  onValueChange,
   format,
   precision,
   minValue,
 }) => (
   <CommonForm label={label} className={`common-numericInput ${className}`}>
-    <NumericInput
+    <ReactNumericInput
       format={format}
       min={minValue}
       value={value}
       precision={precision}
-      onChange={onChange}
+      onChange={onValueChange}
     />
   </CommonForm>
 );
 
-TextInput.propTypes = {
+NumericInput.propTypes = {
   label: PropTypes.string,
   className: PropTypes.string,
   value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   format: PropTypes.func,
   precision: PropTypes.number,
   minValue: PropTypes.number,
-  onChange: PropTypes.func,
+  onValueChange: PropTypes.func,
 };
 
-TextInput.defaultProps = {
+NumericInput.defaultProps = {
   label: '',
   className: '',
   value: 0,
   format: null,
   precision: 0,
   minValue: 0,
-  onChange: noop,
```

Comment:
I am not aware of another usage through the codebase, but if there is one, maybe would be better to keep both onChange and onValueChange?

Reply:
I would prefer to remove them so we won't find ourselves maintaining 2 interfaces.

If there are use cases in plugins where they are using the `onChange` let's deprecate it and remove it next 2 version.

Reply:
Ok, so as I haven't found any other use, but I have just deprecate it anyway, because:
1. we could find a use for the event in the future
2. I could not look good
3. plugins could use it